### ISSUE TITLE
How to add users?

### ISSUE THREAD

**Issue by olendorf** [olendorf/grenada_archaeology#6]
This should be easily doable on the User page. Make sure you are logged in as admin. Then click the "users" button then you should see "New User"

**Comment by hannaj5** [comment]
Yeah, sorry-- that doesn't work for some reason. I can see the form and put in a new email, but nothing happens when I hit "save" or "save new"

**Comment by olendorf** [comment]
Let me look into that.

**Comment by olendorf** [comment]
@hannaj5  OK. I fixed that one. Dumb error on my part, I forgot to add the password and confirmation, which are required. For now you will need to add passwords for people you are registering. This isn't really great, and we can change this easily in the future.

You can incorporate this into your project using the following commands on the terminal in cloud 9.

```
# First make sure all your changes are committed
$ git add -A 
$ git commit -m "useful message"

# If you aren't working in develop branch do this, 
# Basically move to develop branch and merge your previous changes
$ git checkout develop
$ git merge <your previous branch name>

# Pull in the changes in github to your cloud 9
$ git pull origin develop
